service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapperTest.java

```diff
@@ -149,46 +169,142 @@ public void setUp() throws IOException {
             new StructuredObservationValueMapper(),
             new CodeableConceptCdMapper(),
             new ParticipantMapper(),
-            multiStatementObservationHolderFactory
+            multiStatementObservationHolderFactory,
+            confidentialityService
         );
     }
 
     @AfterEach
-    public void tearDown() {
+    void tearDown() {
         messageContext.resetMessageContext();
     }
 
     @ParameterizedTest
     @MethodSource("resourceFileParams")
-    public void When_MappingObservationJson_Expect_CompoundStatementXmlOutput(String inputJson, String outputXml) throws IOException {
-        when(randomIdGeneratorService.createNewId()).thenReturn("random-unmapped-id");
+    void When_MappingObservationJson_Expect_CompoundStatementXmlOutput(String inputJson, String outputXml) {
+        final Observation observationAssociatedWithSpecimen = getObservationResourceFromJson(inputJson);
+        final String expectedXml = getXmlStringFromFile(outputXml);
 
-        String jsonInput = ResourceTestFileUtils.getFileContent(inputJson);
-        Observation observationAssociatedWithSpecimen = new FhirParseService().parseResource(jsonInput, Observation.class);
-        String expectedXmlOutput = ResourceTestFileUtils.getFileContent(outputXml);
+        when(randomIdGeneratorService.createNewId())
+            .thenReturn("random-unmapped-id");
 
-        String compoundStatementXml = observationMapper.mapObservationToCompoundStatement(
-            observationAssociatedWithSpecimen
-        );
+        final String actualXml = observationMapper.mapObservationToCompoundStatement(
+            observationAssociatedWithSpecimen);
 
-        assertThat(compoundStatementXml).isEqualToIgnoringWhitespace(expectedXmlOutput);
+        assertThat(actualXml).isEqualToIgnoringWhitespace(expectedXml);
     }
 
     @Test
-    public void When_MappingDefaultObservationJson_Expect_DefaultObservationStatementXmlOutput() throws IOException {
-        String jsonInput = ResourceTestFileUtils.getFileContent(
-            OBSERVATION_TEST_FILE_DIRECTORY + "input_default_observation.json"
-        );
-        Observation observationAssociatedWithSpecimen = new FhirParseService().parseResource(jsonInput, Observation.class);
-        String expectedXmlOutput = ResourceTestFileUtils.getFileContent(
-            OBSERVATION_TEST_FILE_DIRECTORY + "expected_output_default_observation.xml"
-        );
+    void When_MappingDefaultObservationJson_Expect_DefaultObservationStatementXmlOutput() {
+        final Observation observationAssociatedWithSpecimen =
+            getObservationResourceFromJson("input_default_observation.json");
```

Comment:
Again, having this as one-liner would be much more readable (in my opinion) but it can be subjective, hence I leave it as an option for you to decide.

Reply:
Hey @ole4ryb, 

The reason for splitting these were due to check style constraints - I think keeping these consistent could be
good - but happy to hear your opinions! 😄